libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -39,12 +41,12 @@ bool CompilationResultIsSuccess(const shaderc::CompilationResult& result) {
 // magic number in the fixed postion of the byte array in the result object.
 // Returns true if the magic number is found at the correct postion, otherwise
 // returns false.
-bool IsValidSpv(const shaderc::CompilationResult& result) {
+template <typename T>
+bool IsValidSpv(const shaderc::CompilationResult<T>& result) {
```

Comment:
I think we can use "SpvCompilationResult" instead of "CompilationResult<T>" here. Otherwise the "uint32_t*" below may look a little bit weird.